css/styles.css

```
@@ -169,13 +202,21 @@ body {
   padding: 8px 16px;
 }
 
+.navigation__link:last-child {
+  padding-right: 0;
+}
+
 .navigation__link:hover {
   color: #333;
 }
 
 .section-full-width {
   width: 100%;
-  padding: 64px 48px;
+  padding: 120px;
+}
+
+.section-como-participar {
+  padding-top: 180px;
```

Nesta parte aqui eu sugiro colocar desta outra forma:
```suggestion
  margin-top: 60px;
  padding-top: 120px;
```
A diferença é o espaço que fica na parte superior da seção "Como Participar" quando o link do menu "Como Participar" é acionado.